index.html

```diff
@@ -0,0 +1,237 @@
+          <div class="community-links__article">
+            <div class="community-links__image">image</div>
+            <div class="community-links__title"></div>
+            <div class="community-links__text"></div>
+          </div>
+        </div>
+        <div class="community-links__row">
+          <div class="community-links__article">
+            <div class="community-links__image">image</div>
+            <div class="community-links__title"></div>
+            <div class="community-links__text"></div>
+          </div>
+          <div class="community-links__article">
+            <div class="community-links__image">image</div>
+            <div class="community-links__title"></div>
+            <div class="community-links__text"></div>
+          </div>
+        </div>
+      </div>
+      <div class="featured">
+        <div class="membership__textContainer">
+          <div class="membership__text"></div>
+          <div class="membership__text"></div>
+          <div class="membership__text"></div>
+          <div class="membership__text"></div>
+          <div class="membership__text --membershipButton">Action Button</div>
+        </div>
+        <div class="membership__floatingButton">Floating Button</div>
+      </div>
+      <div class="curated-posts">
+        <div class="general-articles__article">
+          <div class="articleTitle"></div>
+          <div class="articleParagraph"></div>
+        </div>
+        <div class="general-articles__article">
+          <div class="articleTitle"></div>
+          <div class="articleParagraph"></div>
+        </div>
+        <div class="general-articles__article">
+          <div class="articleTitle"></div>
+          <div class="articleParagraph"></div>
+        </div>
+        <div class="general-articles__article">
+          <div class="articleTitle"></div>
+          <div class="articleParagraph"></div>
+        </div>
+      </div>
+      <div class="newsletter">
+        <div class="newsletter__text"></div>
+        <div class="newsletter__text"></div>
+        <div class="newsletter__text"></div>
+        <div class="newsletter__Img">Image</div>
+        <div class="newsletter__inputAction">Input-Action</div>
+      </div>
+    </main>
+    <footer>
+      <div class="footerContent">
+        <div class="footerContent__text --smallTxt"></div>
+        <div class="footerContent__text"></div>
+        <div class="footerContent__text"></div>
+        <div class="footerContent__text"></div>
+        <div class="footerContent__text"></div>
+        <div class="footerContent__blockParent">
+          <div class="footerContent__block">Text</div>
+        </div>
+        <div class="footerContent__blockParent">
+          <div class="footerContent__text --smallTxt"></div>
+        </div>
+        <div class="footerContent__blockParent">
+            <div class="footerContent__text --smallTxt"></div>
+          </div>
+        <div class="footerContent__image">Image</div>
```

Comment:
wrap your footer content in ```<a>``` tags. they should be links. add some text like ```link-1``` so they can be visible.

Reply:
Thanks, we have added `<a>` tags and a word `links` in each line. Its that ok?
![image](https://user-images.githubusercontent.com/36519478/66088860-711f7500-e542-11e9-9b9c-e5be4506efa8.png)